docker-compose.yml

```diff
@@ -1,7 +1,11 @@
 services:
   gulp:
     profiles: ["full"]
+    # image: mentat-is/gulp:latest
     image: gulp:latest
+    # build:
+    #   context: .
+    #   dockerfile: Dockerfile.alpine
```

Comment:
^^^ this is just an example of how to run the compose with a local image, please let me know if you want to remove these comments